src/vos/tests/evt_ctl.c

```diff
@@ -2395,17 +2293,46 @@ ts_cmd_run(char opc, char *args)
 		rc = 0;
 		break;
 	}
-	if (rc != 0)
-		D_PRINT("opc=%d failed with rc="DF_RC"\n", opc, DP_RC(rc));
-
+	if(st != NULL) {
+		rc = -1;
+	}
 	return rc;
 }
 
+static void
+ts_group(void ** state){
+	int	opc = 0;
+	while((opc = getopt_long(test_group_stop-test_group_start+1,
+				 test_group_args+test_group_start,
+				 "C:a:m:e:f:g:d:b:Docl::ts",
+				 ts_ops, NULL)) != -1){
+		ts_cmd_run(opc, optarg);
+	}
+}
+
+static int
+run_cmd_line_test(char* test_name, char** args, int start_idx, int stop_idx)
```

Comment:
(style)  "foo* bar" should be "foo *bar"
(style)  "foo** bar" should be "foo **bar"